### ISSUE TITLE
[css-containment] Reorganizing the Containment specs

### ISSUE THREAD

**Issue by frivoal** [w3c/csswg-drafts#10433]
It seems to me that Container Queries would be more suited to be a standalone spec. It has significant normative dependencies onto CSS-Contain, but isn't about defining containment itself.

I would suggest:
* moving Container Queries to a separate spec, either standalone (probably), or as part of one of the newer levels of `css-conditional`
* moving `inline-size` containment from level 3 to level 2
(this makes contain-3 empty)

While we're at it, I wonder if something similar should happen to `content-visibility` being pulled into its own spec. The same arguments seem to apply, although maybe not as strongly. Alternatively, `content-visibility` is what becomes level 3, and we let level 2 be only about containment in the strict sense (it would differ from level 1 by adding `style` and `inline-size` containment).

I think that would help with stabilizing the core part of containment faster, and would reflect the de-facto division of labor better.

Co-editors (@mirisuzanne @tabatkins @vmpstr ), what do you think?

**Comment by frivoal** [comment]
Btw, https://lists.w3.org/Archives/Public/www-style/2023Nov/0004.html resolved that we should start a css-contain-4 spec for State Container Queries. This document has not been created yet, but if we go ahead with what I proposed above, that would become Level n+1 of wherever we move Container Queries to.

**Comment by mirisuzanne** [comment]
I think it's strange that we are alternating levels for fairly distinct features - so yes, I agree it would be helpful to split CQs into a different module.

**Comment by tabatkins** [comment]
I don't have a strong opinion either way here. Splitting out CQs into their own track sounds reasonable.

**Comment by frivoal** [comment]
There is currently no `css-contain-4` spec, but we do have it as a label for features deferred from level 3. Should I rename that label to `css-conditional-6`?

**Comment by frivoal** [comment]
Should we republish css-contain-3 as a [discontinued draft](https://www.w3.org/policies/process/#abandon-draft), now that it's empty?

**Comment by mirisuzanne** [comment]
Should we also update the working draft of css-conditional-5, so that this change doesn't result in container queries reverting from WD to ED?

**Comment by mirisuzanne** [comment]
@astearns I think we need two publishing resolutions here. It's a bit strange that in the meantime, Container Queries have been entirely removed from any public working draft.

**Comment by svgeesus** [comment]
> Should we also update the working draft of css-conditional-5,

Yes, certainly. The previous TR publication was [21 December 2021](https://www.w3.org/TR/2021/WD-css-conditional-5-20211221/) and we have [a ton of changes since then](https://github.com/w3c/csswg-drafts/commits/main/css-conditional-5/Overview.bs).  The [changes list](https://drafts.csswg.org/css-conditional-5/#changes) is not at all up to date. I will work on updating it, so that it is ready for publication.

**Comment by svgeesus** [comment]
> Should we republish css-contain-3 as a [discontinued draft](https://www.w3.org/policies/process/#abandon-draft), now that it's empty?

Yes, because the changes list is not empty, and those earlier changes are pointed to from css-contain-5.

Is there any downside to making it discontinued, if we want to later add stuff again? I don't recall that I published a discontinued document before.

**Comment by svgeesus** [comment]
[Conditional 5 changes up to date](https://github.com/w3c/csswg-drafts/commit/53a6c69c281e7c2326398cc5d309a9f62bc0e4fb)

**Comment by mirisuzanne** [comment]
Closing this, as I think it's done and published. @frivoal am I missing anything?
